packages/main/src/modules/protocol.ts

```diff
@@ -15,29 +36,11 @@ export function initProtocol() {
     app.setAsDefaultProtocolClient(PROTOCOL);
   }
 
-  async function handleProtocol(protocol: string | undefined) {
-    if (protocol) {
-      log.info('[Main Window] Protocol Handling', protocol);
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      (global as any).protocol = protocol;
-    }
-  }
-
   // Check for protocol in argv immediately if we're already ready
-  if (app.isReady() && process.argv.length > 0) {
+  if (process.argv.length > 0) {
```

Comment:
this `initProtocol` method is called inside the `app.whenReady` callback in the `index.ts`, for that reason is no longer required to verify here if the app is ready